samples/word/01-basics/basic-api-call.yaml

```diff
@@ -13,13 +12,10 @@ script:
         async function run() {
             try {
                 await Word.run(async (context) => {
-                    const range = context.document.getSelection();
+                    let range = context.document.getSelection();
                     range.font.color = "red";
-                    range.load("text");
```

Comment:
I think it's useful showing the load pattern as well.  Esp. to contrast with the Promise-based JS-only version